media/Config/autoCompletePath.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// autoCompletePath make output_path based on input_path and copy former output_path to next input_path
```

Comment:
This comment is a bit hard to understand the meaning.

Reply:
for example, input_path is `foldername/tensorflow.pb` then output_path will be `folername/tensorflow.circle` in import.
and then in optimize, input_path have to be same with import's output_path so input_path will be `fordername/tensorflow.circle`

Reply:
It would be nice if you could add this into the source.